agent/src/beerocks/slave/CMakeLists.txt

```diff
@@ -34,3 +34,33 @@ target_link_libraries(${PROJECT_NAME} rt dl bcl btl tlvf elpp btlvf bwl bpl ${LI
 
 # Install
 install(TARGETS ${PROJECT_NAME} DESTINATION ${CMAKE_INSTALL_BINDIR})
+
+# Tests
+if (BUILD_TESTS)
+    set(TEST_PROJECT_NAME ${PROJECT_NAME}_unit_tests)
+    set(unit_tests_sources
+        gate/unit_test/gate_test.cpp
```

Comment:
Directory name for unit tests in other projects has been named `unit_tests`

Reply:
I checked other directories... missed that `s`.
Fixed.